### PR TITLE
Adds ACLK troubleshooting info 

### PR DESCRIPTION
**Pull request by underhood**
##### Summary
Adds troubleshooting info for documentation.

Please comment (any information you would add?, improve broken english?, ...). @joelhans this should be a good point to start from (can be extended and improved of course).

@cakrit ping

### CODE REVIEW

**Comment by cakrit** [comment]
LGTM!

**Comment by joelhans** [comment]
@cakrit Could you review once more? I updated the text a bit.

**Comment by joelhans** [comment]
@underhood I also think this needs a rebase to remove the Netlify errors.

File: aclk/README.md [hunk 1]
```diff
@@ -23,3 +23,62 @@ Parameter proxy can take one of the following values:
...
+### `aclk-available` is `false`
+
+If `aclk-available` is `false` and all other keys are `true`, your Agent is having trouble connection to the Cloud
+through the ACLK.
+
+Please check your firewall and Netdata proxy settings. If you are sure firewall and proxy settings are not the issue,
+you should consult the Agents `error.log` at `/var/log/netdata/error.log` and contact us by [creating an issue on
+GitHub](https://github.com/netdata/netdata/issues/new?labels=bug%2C+needs+triage%2C+ACLK&template=bug_report.md&title=The+installer+failed+to+prepare+the+required+dependencies+for+Netdata+Cloud+functionality) with details about your system and relevant output from `error.log`.
+
+One common cause of the installer failing to build Cloud features is not having one of the following dependencies on your system:
```
Comment (cakrit) [inline aclk/README.md:81]: Move the lines from here to inside the `cloud available` is `false` section.

I would also remove the back-quotes from the headings completely, special characters there is asking for trouble (@joelhans knows)

Comment (joelhans) [reply aclk/README.md]: Done.

**Comment by underhood** [comment]
This got merged into another PR #8724 and will be merged at once with other ACLK/Cloud docu partial PRs

**Pull request closed without merging**